src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.ps1

```diff
@@ -37,8 +37,50 @@ function Test-AzureRmIotHubLifecycle
 	# Create or Update Resource Group
 	$resourceGroup = New-AzureRmResourceGroup -Name $ResourceGroupName -Location $Location 
 
+	Write-Debug " Create new eventHub " 
+	$namespaceName = "IotHubPSEHNamespaceTest"
```

Comment:
You should not hard code nems in the tests, you should generate names using test common code.  This allows tests to be used against any subscription